appveyor.yml

```diff
@@ -2,21 +2,28 @@ version: 1.0.{build}
 branches:
   only:
   - windows_port
+clone_depth: 5
 os: Visual Studio 2015
 install:
 - cinst cmake.portable wget 7zip.commandline
-- wget --no-check-certificate https://01.org/sites/default/files/libxml2-x86_64-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
-- wget --no-check-certificate https://01.org/sites/default/files/libxml2-x86_64-debug-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
-- wget --no-check-certificate https://01.org/sites/default/files/asio-1.10.6.tar.gz
-- 7z x libxml2-x86_64-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
-- 7z x libxml2-x86_64-debug-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
-- 7z x asio-1.10.6.tar.gz
-- 7z x asio-1.10.6.tar
+- if not exist libxml2-x86_64 (
+    wget --no-check-certificate https://01.org/sites/default/files/libxml2-x86_64-3eaedba1b64180668fdab7ad2eba549586017bf3.zip &
+    7z x libxml2-x86_64-3eaedba1b64180668fdab7ad2eba549586017bf3.zip)
```

Comment:
What do you what to protect from ?
 - If it is from transfer corruption, 7zip will protect from that
 - If it is from a attacher, then it would be useful.

Nevertheless I did not find a way of easly do the check. wget does not have an such option.

Thus I will not do it as:
 - it has nothing to do with seeding up the setup
 - it is not trivial
 - such attach would not be very dangerous.